src/keyboard/Key.tsx

```diff
@@ -53,35 +68,37 @@ export const Key = ({
 }: PropsWithChildren<KeyProps>) => {
   const size = makeSize(props, oneU);
 
-  const children = Children.map(props.children, (c) => (
-    <div
-      data-zoomer={hoverZoom}
-      className="justify-self-center self-center row-start-2 row-end-3 col-start-2 col-end-3 font-keycap text-lg data-[zoomer=true]:group-hover:text-2xl"
-    >
-      {c}
-    </div>
-  ));
-
   return (
     <div
-      className="group inline-flex b-0 justify-content-center items-center transition-all duration-100 data-[zoomer=true]:hover:translate-y-[calc(-1em-7px)] data-[zoomer=true]:hover:translate-x-[calc(-1em)]"
+      className={cn(
+        "group justify-content-center group items-center select-none",
+        "transition-transform origin-center data-[zoomer=true]:hover:scale-[2] data-[zoomer=true]:hover:z-20",
+      )}
```

Comment:
I'm really confused. Is this an extra function call just to avoid the long line?

Reply:
fixed. `cn` was a utility to merge className strings, alongside `twMerge`, however this is an incorrect use case for that utility.